pytorch_lightning/utilities/cli.py

```diff
@@ -34,6 +43,85 @@
     ArgumentParser = object
 
 
+class Registry(UserDict):
+    def __call__(
+        self,
+        cls: Optional[Type] = None,
+        key: Optional[str] = None,
+        override: bool = False,
+    ) -> Callable:
+        """
+        Registers a class mapped to a name.
+
+        Args:
+            cls: the class to be mapped.
+            key : the name that identifies the provided class.
+        """
+        if key is None:
+            key = cls.__name__
+        elif not isinstance(key, str):
+            raise TypeError(f"`key` must be a str, found {key}")
+
+        if key in self and not override:
+            raise MisconfigurationException(f"'{key}' is already present in the registry. HINT: Use `override=True`.")
+
+        def do_register(key, cls) -> Callable:
+            self[key] = cls
+            return cls
+
+        do_register(key, cls)
+
+        return cls
+
+    def register_package(self, module: ModuleType, base_cls: Type) -> None:
+        """This function is an utility to register all classes from a module."""
+        for _, cls in inspect.getmembers(module, predicate=inspect.isclass):
+            if issubclass(cls, base_cls) and cls != base_cls:
+                self(cls=cls)
+
+    def available_objects(self) -> List:
+        """Returns a list of registered objects"""
+        return list(self.keys())
+
+    def __str__(self) -> str:
+        return "Registered objects: {}".format(", ".join(self.keys()))
+
+
+CALLBACK_REGISTRIES = Registry()
+CALLBACK_REGISTRIES.register_package(pl.callbacks, pl.callbacks.Callback)
```

Comment:
```suggestion
CALLBACK_REGISTRY = Registry()
CALLBACK_REGISTRY.register_package(pl.callbacks, pl.callbacks.Callback)
```
Should be singular everywhere

Reply:
Done !